app/Http/Controllers/TwitterBot.php

```diff
@@ -102,4 +105,31 @@ private function filterPaymentTweet($payment)
             ', Payed The Sum of ₦'.$amount." to ".$benefactor.' '.$company_handle.' for the '.$description;
         return $tweet;
     }
+
+    public function budgetTweet()
+    {
+        $budget = Budget::inRandomOrder()->first();
+        $tweet = $this->budgetTweetFilter($budget);
+        return $tweet;
+    }
+    public function budgetTweetFilter($budget)
+    {
+        $code = explode('-', $budget->code);
+        if (count($code) > 1) {
+            $ministry_code = $code[1];
+            $sector_code = $code[0];
+            $ministry = Ministry::where('shortname', 'LIKE', "{$ministry_code}%")->first();
+            $sector = Sector::whereId($sector_code)->first();
+            if (is_null($sector)) {
+                return 'the amount of ₦'.$budget->amount." was allocated for ".$budget->classification." in the ".$budget->year ." budget";
```

Comment:
Also try to start the sentences with Uppercase.

Reply:
okay